classes/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] A method for adding rentals should be present. Please Implement it.
- [ ] The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. ie:
`Rental.new(date, self, person)`